### PR TITLE
Load assemblies in .NET Core

### PR DESCRIPTION
**Pull request by Jmales**

### CODE REVIEW

File: WebViewControl.Avalonia/AssemblyCache.NETCore.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿using System.Reflection;
+using System.Runtime.Loader;
+
+namespace WebViewControl {
+
+    partial class AssemblyCache {
```
Comment (joaompneves) [inline WebViewControl.Avalonia/AssemblyCache.NETCore.cs:6]: AssemblyLoader

File: WebViewControl/AssemblyCache.cs [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 namespace WebViewControl {
 
-    internal class AssemblyCache {
+     internal partial class AssemblyCache {
```
Comment (joaompneves) [inline WebViewControl/AssemblyCache.cs:8]: leave as it was

File: WebViewControl/AssemblyCache.cs [hunk 3]
```diff
@@ -42,7 +42,8 @@ internal class AssemblyCache {
                 if (assembly == null) {
                     try {
                         // try load assembly from its name
-                        assembly = AppDomain.CurrentDomain.Load(new AssemblyName(assemblyName));
+                        var fullAssemblyName = Path.Combine(Path.GetDirectoryName(new Uri(Assembly.GetExecutingAssembly().CodeBase).LocalPath), assemblyName + ".dll");
```
Comment (joaompneves) [inline WebViewControl/AssemblyCache.cs:45]: Path.Combine(AppDomain.CurrentDomain.BaseDirectory, assemblyPath + ".dll")

File: WebViewControl.Avalonia/AssemblyLoader..NETCore.cs [hunk 4]
```diff
@@ -0,0 +1,11 @@
+﻿using System.Reflection;
+using System.Runtime.Loader;
+
+namespace WebViewControl {
+
+    internal class AssemblyLoader {
```
Comment (joaompneves) [inline WebViewControl.Avalonia/AssemblyLoader..NETCore.cs:6]: static

File: WebViewControl.Avalonia/AssemblyLoader..NETCore.cs [hunk 5]
```diff
@@ -0,0 +1,11 @@
+﻿using System.Reflection;
```
Comment (joaompneves) [inline WebViewControl.Avalonia/AssemblyLoader..NETCore.cs:1]: is there a double dot in the file name?

File: WebViewControl/AssemblyLoader.NETFramework.cs [hunk 6]
```diff
@@ -0,0 +1,10 @@
+﻿using System.Reflection;
+
+namespace WebViewControl {
+
+    internal class AssemblyLoader {
```
Comment (joaompneves) [inline WebViewControl/AssemblyLoader.NETFramework.cs:5]: static

File: WebViewControl.Avalonia/AssemblyLoader..NETCore.cs (hunk 5 above)
Comment (Jmales) [reply WebViewControl.Avalonia/AssemblyLoader..NETCore.cs]: Oops...

**Pull request merged**